astropy/units/format/generic.py

```diff
@@ -148,7 +148,9 @@ def t_FUNCNAME(t):
             return t
 
         def t_UNIT(t):
-            r"%|([YZEPTGMkhdcmunpfazy]?'((?!\d)\w)+')|((?!\d)\w)+"
+            "%|([YZEPTGMkhdcmu\N{MICRO SIGN}npfazy]?"+r"'((?!\d)\w)+')|((?!\d)\w)+"
```

Comment:
Should remain a raw string, right?
```suggestion
            r"%|([YZEPTGMkhdcmu\N{MICRO SIGN}npfazy]?"+r"'((?!\d)\w)+')|((?!\d)\w)+"
```

Reply:
No, in the first part I need `\N` to be interpreted correctly, but for the later parts one wants to keep `\d` etc. This is why I'm adding a regular to a raw string...

Reply:
Okay, sorry for the noise!